tests/e2e/util.go

```diff
@@ -2360,17 +2404,55 @@ func getPvFromSupervisorCluster(pvcName string) *v1.PersistentVolume {
 	return svcPV
 }
 
-func verifyFilesExistOnVSphereVolume(namespace string, podName string, filePaths ...string) {
+// verfiy File exists or not in vSphere Volume
+func waitAndVerifyFilesExistOnVSphereVolume(namespace string, podName string, poll, timeout time.Duration, filePaths ...string) {
```

Comment:
I meant for you to keep verifyFilesExistOnVSphereVolume() and write a wrapper for polling function